docs/gpa.md

```diff
@@ -378,7 +378,8 @@ If you are interested in using OCDS to publish this, please contact <data@open-c
         <td markdown=1>
 
 1. [Create an OCDS release](../operations/#create-a-release)
-1. Add 'tenderUpdate' to the `tag` array
+1. Set the `tag` array:
+  1. If data has been published acc
```

Comment:
This seems incomplete?

Reply:
I had tried to write guidance to cover all the scenarios described in my comment: https://github.com/open-contracting-extensions/government-procurement-agreement/pull/51#discussion_r524402113 but forgot to remove it when I commited.